src/main/java/org/spongepowered/plugin/meta/PluginMetadata.java

```diff
@@ -48,6 +48,7 @@
 
     @Nullable private String description;
     @Nullable private String url;
+    @Nullable private String mcVersion;
```

Comment:
Can you change this to `minecraftVersion`? The field names do not strictly follow the mcmod.info naming